synapse/replication/http/account_data.py

```diff
@@ -48,14 +52,18 @@ def __init__(self, hs: "HomeServer"):
         self.clock = hs.get_clock()
 
     @staticmethod
-    async def _serialize_payload(user_id, account_data_type, content):
+    async def _serialize_payload(  # type: ignore[override]
```

Comment:
Each instance of this have different arguments, defined as `kwargs` in the super-class. Not sure if there's something better to do than ignore the override?